modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,3 +1,27 @@
 = What’s New in Studio
```

Comment:
Are you sharing this content with the What's New section in the Release Notes? It would be good to set up a partial for this.

Reply:
No, the information included in this What's New page is not the same included in the RNs. This article includes more details. The version in the What's New section of the RN is shorter than this version.

Reply:
I think they should be the same.